cpp/ql/src/semmle/code/cpp/models/implementations/StdString.qll

```diff
@@ -13,6 +20,25 @@ class StdStringCStr extends TaintFunction {
   }
 }
 
+/**
+ * The `std::string` function `operator+`.
+ */
+class StdStringPlus extends TaintFunction {
```

Comment:
Can you extend `AllocationFunction` here as well?

Reply:
just looked at the spec again and I think I originally misread the allocator for the hidden storage as being an allocator for the return value